docs/source/extending/advanced-users.md

```diff
@@ -1,12 +1,12 @@
 # Advanced User Tour
 
-Olive provides simple  Python and command line interface to optimize the input model. See [Quick Tour](../overview/quicktour.md) for more information.
+Olive provides simple Python and command line interface to optimize the input model.
 ```bash
 olive run --config user_provided_info.json
 ```
 
 ```python
-from olive.workflows import run as olive_run
+from olive.workflows import run as olive_run`
```

Comment:
Was this "`" accidentally added? Also fails doc build because of this https://dev.azure.com/aiinfra/PublicPackages/_build/results?buildId=613568&view=logs&j=10899750-8618-5191-93c7-9e3d63cd5bc7&t=8ed8d77a-c8e7-57f0-faf2-09d0e9419c33&l=472

Reply:
Fixed